src/App.tsx

```diff
@@ -46,6 +50,24 @@ const App: React.FC = () => {
     } catch (error) {
       alert('Error' + error);
     }
+
+    // Languages
+    try {
+      const res = await fetch(`https://api.github.com/repos/${owner}/${repo}/languages`);
+      const resJSON = await res.json();
+      console.log(resJSON);
+    } catch (error) {
+      alert('Error' + error);
+    }
+
+    // Contributors
+    try {
+      const res = await fetch(`https://api.github.com/repos/${owner}/${repo}/contributors`);
+      const resJSON = await res.json();
```

Comment:
@mohammedsahl Can you check if you can specify JSON return object so we don't need to cast it everytime?
The Fetch command takes in application/type IF we put it to JSON should return a JSON back / clean up code?

Reply:
Not exactly clear on what you mean by specifying a JSON return object. Do you mean specifying a header in the get request that would indicate that a JSON object must be returned?

Reply:
I think that that's what he's talking about. That would allow you to avoid having to call `.json()` on the response body you get back from your `fetch()` call.

I don't think messing with your `fetch()` call to do that is necessary though. Granted, I haven't worked with APIs and JavaScript very much, but from what I have seen, calling `.json()` on a response body that you get back from an HTTP API call is standard/normal, and appears a lot in people's "API call workflow" for a front-end project.